assets/index.html

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="description"
+      content="InfluxDB is a time series database designed to handle high write and query loads.
+      Telegraf. Open source server agent to collect metrics from stacks, sensors and systems.
+      InfluxDB Enterprise. Turns any InfluxData instance into a production-ready cluster that can run anywhere."
```

Comment:
I think this might be a better choice: view-source:https://www.influxdata.com/products/influxdb-cloud/

>InfluxDB Cloud provides the fastest way to access the most powerful time series database as a service. InfluxDB Cloud is much more than just a database.